app/controllers/application_controller.rb

```diff
@@ -2,4 +2,12 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :configure_permited_parameters, if: :devise_controller?
+  protected
+
+  def configure_permited_parameters
+    devise_parameter_sanitizer.for(:account_update) do |u|
+      u.permit(:bio, :name, :email, :password, :password_confirmation, :current_password)
```

Comment:
Hmmm, so you need to override all of the attributes instead of just adding the ones you need. Seems like there should be a way around this, but go ahead and lleave it for now unless someone else knows the fix when they are reviewing.